zebra/zebra_mroute.h

```diff
@@ -29,7 +29,9 @@ extern "C" {
 #endif
 
 struct mcast_route_data {
-	struct prefix_sg sg;
+	int family;
+	struct prefix src;
+	struct prefix grp;
```

Comment:
This should use `struct ipaddr` here since src/grp will always be full 32/128 prefix length. Making it `struct prefix` gives the mistaken impression that maybe we can make a call to get statistics for a range of addresses.

Reply:
Addressed